Cargo.toml

```diff
@@ -178,6 +178,7 @@ phf = "0.11.2"
 pico-args = "0.5.0"
 prettyplease = "0.2.25"
 project-root = "0.2.2"
+quick-junit = "0.5.1"
```

Comment:
I do not like that we need a dependency just for a output format. 
You could probably self build the XML, like we did with the checkstyle format:
https://github.com/oxc-project/oxc/blob/e929f268260b10e6ec0076bef2b150b0cb1469f3/apps/oxlint/src/output_formatter/checkstyle.rs#L40-L67

Reply:
Done. Removed quick-junit dependency.